pyproject.toml

```diff
@@ -3,3 +3,71 @@ build-backend = 'setuptools.build_meta'
 # https://github.com/yt-dlp/yt-dlp/issues/5941
 # https://github.com/pypa/distutils/issues/17
 requires = ['setuptools > 50']
```

Comment:
This might need to be bumped to 61 (See note in the [docs](<https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html>))